back-end/app/Http/Controllers/SectionController.php

```diff
@@ -17,13 +17,42 @@ public function store(Request $request)
         ]);
 
         Sections::create([
+
             'name' => $request['name']
 
         ]);
 
         return response()->json(["message" => "Section added successfully"], 200);
     }
 
+    public function show()
+    {
+        return response()->json(Sections::all(), 200);
+    }
+
+    public function update(Request $request, $sectionId)
+    {
+        $request->validate([
+            'name' => 'required|string',
+        ]);
+
+        Sections::where('id', $sectionId)->update(['    name' => $request['name']]);
+
+        return response()->json(["message" => "Project updated successfully"], 200);
+    }
+
+
+    public function delete($sectionId)
+    {
+        if (SectionUser::where('sectionId', $sectionId)->exists()) {
+            return response()->json(["message" => "Section still has users assigned"], 400);
```

Comment:
```suggestion
            return response()->json(["message" => "Section still has users assigned"], 409);
```

Makkelijker op te vangen vanuit frontend. 409 staat voor conflict